src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { getNetworks } from 'src/config'
+import { AddressBookEntry, makeAddressBookEntry } from 'src/logic/addressBook/model/addressBook'
+import { addressBookAddOrUpdate } from 'src/logic/addressBook/store/actions'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { MIGRATION_ADDRESS } from 'src/routes/routes'
+
+const mergeAddressBooks = (addressBookEntries: AddressBookEntry[], dispatch) => {
```

Comment:
This function needs to be unit-tested, esp. the case when you have the same address on two networks.